src/citrine/_serialization/properties.py

```diff
@@ -79,7 +79,10 @@ def deserialize_from_dict(self, data: dict) -> DeserializedType:
         value = data
         fields = self.serialization_path.split('.')
         for field in fields:
-            value = value.get(field, self.default)
+            if isinstance(value, dict):
+                value = value.get(field, self.default)
+            else:
+                value = self.default
```

Comment:
In this instance we're deserialization the field `'report.descriptors'`. If `report` key does not exist in the JSON then it takes the default value, which is an empty list. But it then tries to extract the `descriptors` key from that object, and an empty list does not have a `get` method.

The assumption being made here is that the JSON might have missing terminal fields but its basic structure is intact. That's not true in all cases.

Reply:
Why is the default value of "report" an empty list?  I agree with @asantas93 that this fix is smelly.

Reply:
It's not. The default value of `report.descriptors` is an empty list. `report` isn't a field we deserialize.

Reply:
> If report key does not exist in the JSON then it takes the default value, which is an empty list.

That's what I was commenting on.

Which of the three added tests was the one that was failing?

Reply:
I updated the comment with better prepositions.

They were all failing. `report=None` corresponds to what the backend returns for still-validating predictors.